packages/discord.js/src/managers/ApplicationCommandPermissionsManager.js

```diff
@@ -82,19 +74,25 @@ class ApplicationCommandPermissionsManager extends BaseManager {
    */
 
   /**
-   * Fetches the permissions for one or multiple commands.
+   * Fetches the permissions for one or multiple commands. Provide the clients id as the command id to fetch *only* the
```

Comment:
This sentence sounds confusing overall. Are you providing your bot's ID as the command ID to fetch the permissions on the guild? Doesn't make much sense imo
```suggestion
   * Fetches the permissions for one or multiple commands. Provide the client's id as the command id to fetch *only* the
```

Reply:
Blame discord

Reply:
Well alright if the functionality is well described that’s fine, just the typo then